python-client/get-started/introduction.mdx

```diff
@@ -28,6 +28,8 @@ from literalai import AsyncLiteralClient
 async_literal_client = AsyncLiteralClient(api_key=os.getenv("LITERAL_API_KEY")) 
 ```
 
+Additionally, you can pass the optional `batch_size` argument to batch multiple events - handled simultaneously by Literal. Defaults to 10.
```

Comment:
A bit more explanation what this affects and that you need to flush at the end. Also show how this is actually written in the code, like `literal_client = LiteralClient(api_key=os.getenv("LITERAL_API_KEY"), batch_size=10)` as a line under the line without batch size. 

Perhaps this can even be under a new section with a header like "Batching", just like we have "disable decorators"